src/components/custom/_dbanner.tsx

```diff
@@ -129,7 +129,7 @@ const StyledHeader = styled(Header)`
 `
 const DBanner = ({ title, data, background_pattern, image_alt }: DBannerProps) => {
     const handleSignup = useHandleSignup()
-    const { is_eu, is_row } = useCountryRule()
+    const { is_eu, is_row } = React.useContext(RegionContext)
```

Comment:
Just a suggestion
Instead of repeating this everywhere just create a custom hook and import it from there, sth like this!

custom hook
`const useRegionContext = () => React.useContext(RegionContext)`

```suggestion
    const { is_eu, is_row } = useRegionContext()
```

Reply:
cc: @michio-rework @ali-hosseini-deriv